### ISSUE TITLE
Should app.js be gitignored

### ISSUE THREAD

**Issue by sunscreem** [picandocodigo/efp_discount#1]
After running gulp, git is wanting to commit app.js. 

Is that going to cause conflicts? Is it built on deploy to github pages?

**Comment by picandocodigo** [comment]
Yes, `app.js` is just the minified version and it's in version control to deploy to GitHub pages. One solution would be to have a `master` branch and a script to only deploy the minified version to gh-pages or use some other tool to do that. But since it's such a small project I thought it wasn't worth the effort. In the meantime, if anything's changed in `calculator.js`, then grunt should be ran and `app.js` should be committed. Maybe I could add this to the README.

**Comment by sunscreem** [comment]
Yeah - probably a good plan. I'll forget and break it ;)
